include/slb/type_traits.hpp

```diff
@@ -1023,6 +1026,12 @@ template <std::size_t Len, typename... Ts>
 constexpr std::size_t aligned_union<Len, Ts...>::alignment_value;
 #endif
 
+template <typename T>
+struct remove_cvref {
+  using type =
+      typename std::remove_cv<typename std::remove_reference<T>::type>::type;
```

Comment:
Can we just use `slb::remove_cv_t<slb::remove_reference_t<T>>` here?

Reply:
Maybe, if they are lined up correctly in the header. What would be the point?

Reply:
Yeah, they happen to be in this case. But I guess it's not much of a win.